src/Infrastructure.Dapper/Repositories/AuthRequestRepository.cs

```diff
@@ -13,5 +16,16 @@ public AuthRequestRepository(GlobalSettings globalSettings)
         public AuthRequestRepository(string connectionString, string readOnlyConnectionString)
             : base(connectionString, readOnlyConnectionString)
         { }
+
+        public async Task<int> DeleteExpiredAsync()
+        {
+            using (var connection = new SqlConnection(ConnectionString))
+            {
+                return await connection.ExecuteAsync(
+                    $"DELETE FROM [{Schema}].[AuthRequest] WHERE ExpirationDate < GETUTCDATE();",
```

Comment:
Lets wrap this in a stored proc similar to the other jobs. Please use the existing migration script for this `AuthRequests_Init`.